cmd/skopeo/main.go

```diff
@@ -85,30 +75,47 @@ func createApp() (*cli.App, *globalOptions) {
 			Usage:       "use `VARIANT` instead of the running architecture variant for choosing images",
 			Destination: &opts.overrideVariant,
 		},
-		cli.DurationFlag{
-			Name:        "command-timeout",
-			Usage:       "timeout for the command execution",
-			Destination: &opts.commandTimeout,
+		cli.StringFlag{
+			Name:        "policy",
+			Usage:       "Path to a trust policy file",
+			Destination: &opts.policyPath,
 		},
 		cli.StringFlag{
 			Name:        "registries-conf",
 			Usage:       "path to the registries.conf file",
 			Destination: &opts.registriesConfPath,
 			Hidden:      true,
 		},
+		cli.StringFlag{
+			Name:        "registries.d",
+			Usage:       "use registry configuration files in `DIR` (e.g. for container signature storage)",
+			Destination: &opts.registriesDirPath,
+		},
+		cli.GenericFlag{
+			Name:   "tls-verify",
+			Usage:  "require HTTPS and verify certificates when talking to container registries (defaults to true)",
+			Hidden: true,
+			Value:  newOptionalBoolValue(&opts.tlsVerify),
+		},
+		cli.StringFlag{
+			Name:        "tmpdir",
+			Usage:       "directory used to store temporary files",
+			Destination: &opts.tmpDir,
+			Value:       getTempDir(),
+		},
```

Comment:
Does this flag do much?  If $TMPDIR is set, won't the system already use that?  Should we instead take a value too and use that directory if it's found, and if not, then default to $TMPDIR and then if that's not set, then /var/tmp?

Reply:
No the system does not currently use TMPDIR for this.  This chenage gets it to use the tmpdir and allows users to specify alternative directories.